### DISCUSSION TITLE
请问为什么“这么多”“这么长”的“连接”？meta 内核发现的

### DISCUSSION THREAD

**Discussion by smart-z** [vernesong/OpenClash#3865] [Q&A]
我因为节点关系，vless 只能选 meta 核心，换了 meta 核心后，就发现，如此多如此长的连接就在那里不消失。有时候甚至要 1000 个连接了，长达 6 个小时之久。
这是咋回事？能解决吗？搜了下议题和讨论里面的内容，似乎没有说解决办法。所以我这又发了一个讨论，实在抱歉，我不是程序员不太懂这些。
恳请得到一点帮助，谢谢了，跪谢大神

**Comment by smart-z** [comment]
![CleanShot 2024-04-30 at 07 36 42](https://github.com/vernesong/OpenClash/assets/1166675/b62ef3c9-7e11-4c19-9499-64e0784abdd8)

**Comment by refined-fish** [comment]
持续时间现在可以修改，在meta内核设置里，但不要改的太短，会导致苹果移动设备耗电增高（一直连接苹果服务器）
